server.go

```diff
@@ -274,7 +274,14 @@ func (s *server) channelHandleWrapper(handler ChannelHandler, handlerFunc Channe
 		// get the channel for this request - can be nil, e.g. FBA verification requests
 		channel, err := handler.GetChannel(ctx, r)
 		if err != nil {
-			logrus.WithError(err).WithField("request", string(recorder.Trace.RequestTrace))
+
+			// webhooks for FBA, IG and WAC should always return 200
+			if handler.ErrorResponseStatus() == 200 {
```

Comment:
if we want all places where we can return an error to use 200, then can we update `writeAndLogRequestError` to take the handler and call `ErrorResponseStatus()` ?

Reply:
`writeAndLogRequestError` is in courier package and trying to reference handlers there causes cycle import issue

Reply:
Ok.. we can merge this to get FBA working.. but `handler.ErrorResponseStatus() == 200` is not the greatest bit of code ever written so we should take a pass at trying to refactor how handlers customize responses.